### ISSUE TITLE
How to get it to work?

### ISSUE THREAD

**Issue by landland** [jamauro/pub-sub#6]
I tried this package on two large apps and could not get it going. Basically, the steps for both apps were:

1. meteor add jam:pub-sub
2. change Meteor.publish to Meteor.publish.stream on the server. Also tried Meteor.publish.once but no luck.

```
// isomorphic
Jobs = new Mongo.Collection('jobs');
FlowRouter.route('/jobs', {
    action: function(params, queryParams) {
        this.render("_layout", "jobs");
    },
    fastRender: true,
    waitOn(params, queryParams) {
        return [Meteor.subscribe('processingJobs')];
    }
});
```

```
//server
Meteor.publish.stream('processingJobs', function() {
    if (!this.userId) {
        return this.ready();
    }
    this.unblock();
    return Jobs.find({
        processed: false,
    });
});
```

That's all I changed. Was there something else I needed to do? The apps are huge but this is the gist of it. I am using redis-oplog with changestream-to-redis and have disable-oplog if that matters.

**Comment by jamauro** [comment]
Hmm, do you see any errors anywhere that might give any clues? 

If not, could you put together a minimal repo that I could pull? That would really help with diagnosing the underlying issue.

**Comment by landland** [comment]
There's no error, unfortunately.

The apps are huge, I think a minimal repo is going to be hard. If you don't see anything incorrect with usage and it looks like the code should work on something like a simple todo app, let me try to start removing a few packages to see if that changes anything.

**Comment by jamauro** [comment]
I'm going to close this issue. If you can help with a minimal repo that isn't behaving as you'd expect, feel free to re-open or open a new issue. I've just released `v0.3.0` with several improvements.
